contents/handbook/growth/marketing/in-app.md

```diff
@@ -0,0 +1,41 @@
+---
+title: In-App
+sidebar: Handbook
+showTitle: true
+---
+
+Occasionally, we use in-app messages to tell users about certain things. We recognize that in-app messages can be intrusive and we want to avoid spamming our users with too many of them, too frequently. For that reason, we're judicious about the way in which we use them. 
+
+We currently don't have a separate system for tracking in-app messages, so Marketing currently owns the channel and is responsible for ensuring that messages aren't used excessively.
+
+## Types of in-app message
+
+Currently, there are three ways in which we can send in-app messages.
+
+- **Notification bar:** A message displayed across the top of the page, activated using the [Notification Bar](https://github.com/PostHog/notification-bar-app) app.
+- **In-app prompt:** A customizable pop-up which can be targeted to certain URLs and made to appear in the center of the page, or anchored in the corner. Activated using the `prompt` feature flag. [More info](/handbook/product/in-app-prompts).
+- **Notifications:** A notification which is pushed into the navigation bar, as a number on the bell icon. Activated using the `changelog-notification` feature flag. 
+
+## How we use in-app messages
+
+We use each of the three channels above for different purposes, guided by the needs of a message and the level of intrustion. 
+
+- The notification bar is only used for messages which must be urgently communicated to all users, such as messages about service disruption.
+- In-app prompts can be used for a wide variety of purposes, including promotion of new features. However, users will only see one in-app prompt per day and should be targeted to appear only on relevant pages and to relevant users. In-app prompts shouldn't be used to message all users at once, or to direct users to another part of the app.
+- Notifications can be used for a wide variety of purposes and are minimally intrusive. We regularly use notifications to promote new features via the PostHog changelog. 
+
+## Creating new in-app prompts
+
+In-app prompts are intrusive to users, but can be used for a wide variety of reasons. Therefore, if you create one we ask that you...
+
+1. Raise an issue in [the meta repo](https://github.com/PostHog/meta) describing your in-app prompt and how it will be used. Please explain what content it will show, and how it will be targeted.  
```

Comment:
What will happen with the issues? Closed once complete? I'm just envisioning a future where we have a bunch of open issues for in-app prompts that never go away.